nova_vm/src/ecmascript/types/language/object/property_key.rs

```diff
@@ -52,6 +52,15 @@ impl PropertyKey {
         self.into()
     }
 
+    pub fn from_value(agent: &Agent, value: Value) -> Option<Self> {
```

Comment:
**praise:** This is good. I think I have an issue somewhere (#183) that basically says we should remove the TryFrom and Into impls to convert between PropertyKey and Value/String so freely, since they're effectively unsafe from consistency point of view.

An unsafe method to turn a String into a PropertyKey without checking for integerness is probably worth a consideration, though.